.github/workflows/publish.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish
+
+on:
+  release:
+    types: [published, edited]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.release.tag_name }}
+      - name: Use node.js 14.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14
+      - name: NPM Install
+        run: npm ci
+      - name: Check Format
+        run: npm run format:check
+      - name: Lint
+        run: npm run lint
+      - name: Build
+        run: npm run build
+      - name: Test
+        run: npm run test
+      - uses: JasonEtco/build-and-tag-action@v2
```

Comment:
does this also publish?

Reply:
It creates additonal semver tags on publish

Reply:
What @snowiow, it won't publish a new release. But you can create a new release on the release page and this workflow will pin the major semver version to the latest release commit.